test_project/settings.py

```diff
@@ -236,12 +257,12 @@
 
 CACHE_MIDDLEWARE_KEY_PREFIX = "YAK"
 
-CACHES = {
-    "default": {
-        "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
-        "LOCATION": "localhost:11211",
-    }
-}
+# CACHES = {
```

Comment:
We also might want a ticket to update this to the new preferred way of configuring settings?